ngrinder-controller/src/main/java/org/ngrinder/infra/config/Config.java

```diff
@@ -1,741 +1,748 @@
+				return gitHubConfigTemplate;
+			} catch (IOException e) {
+				LOG.error("Error while load gitconfig-template.yml", e);
+			}
+		}
+		return gitHubConfigTemplate;
+	}
+
+	/**
+	 * Get the internal properties.
+	 *
+	 * @return internal properties
+	 */
+	public PropertiesWrapper getInternalProperties() {
+		return internalProperties;
+	}
+
+
+	/**
+	 * Check if it's verbose logging mode.
+	 *
+	 * @return true if verbose
+	 */
+	public boolean isVerbose() {
+		return verbose;
+	}
+
+	/**
+	 * Get the currently configured controller IP.
+	 *
+	 * @return current IP.
+	 */
+	public String getCurrentIP() {
+		if (cluster) {
+			return StringUtils.trimToEmpty(getClusterProperties().getProperty(PROP_CLUSTER_HOST));
+		} else {
+			return StringUtils.trimToEmpty(getControllerProperties().getProperty(PROP_CONTROLLER_IP));
+		}
+	}
+
+
+	/**
+	 * Check if the current ngrinder instance is hidden instance from the cluster.
+	 *
+	 * @return true if hidden.
+	 */
+	public boolean isInvisibleRegion() {
+		return getClusterProperties().getPropertyBoolean(PROP_CLUSTER_HIDDEN_REGION);
+	}
+
+	/**
+	 * Check if no_more_test.lock to block further test executions exists.
+	 *
+	 * @return true if it exists
+	 */
+	public boolean hasNoMoreTestLock() {
+		return exHome.exists() && exHome.getSubFile("no_more_test.lock").exists();
+	}
+
+	/**
+	 * Check if shutdown.lock exists.
+	 *
+	 * @return true if it exists
+	 */
+	public boolean hasShutdownLock() {
+		return exHome.exists() && exHome.getSubFile("shutdown.lock").exists();
+	}
+
+	/**
+	 * Get the date of the recent announcement modification.
+	 *
+	 * @return the date of the recent announcement modification.
+	 */
+	public Date getAnnouncementDate() {
+		return announcementDate;
+	}
+
+	/**
+	 * Get ngrinder help URL.
+	 *
+	 * @return help URL
+	 */
+	public String getHelpUrl() {
+		return getControllerProperties().getProperty(PROP_CONTROLLER_HELP_URL);
+	}
+
+	public PropertiesWrapper getClusterProperties() {
+		return clusterProperties;
+	}
+
+	public PropertiesWrapper getLdapProperties() {
+		return ldapProperties;
+	}
+
+	/**
+	 * Get the time out milliseconds which would be used between the console and the agent while preparing to test.
+	 *
+	 * @return
+	 */
+	public long getInactiveClientTimeOut() {
+		return getControllerProperties().getPropertyLong(PROP_CONTROLLER_INACTIVE_CLIENT_TIME_OUT);
+	}
+
+	public boolean isEnableStatistics() {
+		return getControllerProperties().getPropertyBoolean(PROP_CONTROLLER_ENABLE_STATISTICS);
+	}
+
+	/**
+	 * Get Csv Separator.
+	 * default csvSeparator Value is comma
+	 *
+	 * @return String csvSeparator
+	 */
+	public String getCsvSeparator() {
+		String csvSeparator = getControllerProperties().getProperty(PROP_CONTROLLER_CSV_SEPARATOR);
+		if (("tab").equals(csvSeparator)) {
+			csvSeparator = "\t";
+		} else if (("semicolon").equals(csvSeparator)) {
+			csvSeparator = ";";
+		} else {
+			csvSeparator = ",";
+		}
+
+		return csvSeparator;
+	}
+
+}
```

Comment:
Why is all file changed? Do u use different line end?

Reply:
It caused by using different line separator. Changed it and force pushed branch again.